source/Cosmos.Build.Tasks/build/Cosmos.Build.targets

```diff
@@ -383,10 +383,12 @@
         <CreateGrubConfig TargetDirectory="$(IntermediateIsoDirectory)"
                               BinName="$([System.IO.Path]::GetFileName('$(BinGzFile)'))"
                               Modules="gzio"
+                              timeout="$(Timeout)"
```

Comment:
Just to confirm it ok not to capitalize the timeout in the call here?

Reply:
Wait I did capitalize that in my code? I'll check it out and see what's going on

Reply:
Problem has been fixed